examples/data_augmentation/tutorial_for_data_augmentation.ipynb

```diff
@@ -16,12 +16,24 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<forte.pipeline.Pipeline at 0x7f95815ae2d0>"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
```

Comment:
Let's clear the output (i.e. make output an empty list `[]`).

Reply:
Sure. I just submitted a new commit.